README.md

```diff
@@ -38,8 +35,11 @@ to listen via HTTPS, then don't use HTTP.
 Build the dockerfile and then run it:
 
 ```sh
-$ docker build . -t http-proxy
-$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 http-proxy
+docker build . -t http-proxy
+# Or build with the metrics feature enabled
+docker build . --build-arg FEATURES="expose-metrics" -t http-proxy
+
+docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 http-proxy
```

Comment:
```suggestion
$ docker build . -t http-proxy
# Or build with the metrics feature enabled
$ docker build . --build-arg FEATURES="expose-metrics" -t http-proxy

$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 http-proxy
```
I'd keep the $ to indicate that this is a shell command and to keep consistency with the part below.